### DISCUSSION TITLE
Developer Documentation

### DISCUSSION THREAD

**Discussion by lukew3** [monkeytypegame/monkeytype#1826] [Ideas]
Should be some sort of developer documentation so that new contributors can get an overview of how the code works so that they can start contributing.

**Comment by tajhans** [comment]
Since so many people have asked for this I think it’s safe to say that we should add some beginner documentation.

**Reply by unrealapex** [reply]
I think the contributing file covers everything pretty well. If you want me to add or revise a section on something, don't hesitate to ask.

**Reply by lukew3** [reply]
@UnrealApex The contributing file covers setup well, but I think that it would be helpful to have some guide that tells developers how to navigate the code and what different files and functions do. This would help developers get a broad understanding of how the project works and where they need to go to work on their issue.

**Reply by unrealapex** [reply]
I guess that would help more than just how to setup and build Monkeytype.
